client/internal/wgproxy/ebpf/wrapper.go

```diff
@@ -42,3 +80,45 @@ func (e *ProxyWrapper) CloseConn() error {
 	}
 	return nil
 }
+
+func (p *ProxyWrapper) proxyToLocal(ctx context.Context) {
+	defer p.WgeBPFProxy.removeTurnConn(uint16(p.wgEndpointAddr.Port))
+
+	buf := make([]byte, 1500)
+	for {
+		n, err := p.readFromRemote(ctx, buf)
+		if err != nil {
+			return
```

Comment:
Let's log the error here as well

Reply:
We log in to the readFromRemote function. I moved there because I reached the limit of the code complexity.